backend/internal/model/handler/http/v1/model.go

```diff
@@ -55,7 +57,7 @@ func NewModelHandler(
 //	@Success		200		{object}	web.Resp{data=domain.CheckModelResp}
 //	@Router			/api/v1/model/check [post]
```

Comment:
Check 方法中使用了新的 ModelKit 实例调用，但需要确保 ModelKit 的 CheckModel 方法签名与之前全局函数的签名一致。

```suggestion
建议添加注释说明方法调用的变更：

```go
// 从 modelkit.CheckModel 改为 h.modelkit.CheckModel
modelkitRes, err := h.modelkit.CheckModel(c.Request().Context(), &modelkitDomain.CheckModelReq{
    // ... 参数
})
```